package.json

```diff
@@ -18,6 +18,7 @@
     "test": "mocha test",
     "lint": "tslint -c tslint.json -p . ./src/*.ts ./src/**/*.ts ./src/**/**/*.ts ./src/**/**/**/*.ts && eslint test",
     "build": "tsc",
+    "nuke": "rm -rf node_modules; rm -rf build && npm run clean",
```

Comment:
Suggest also `rm -f package-lock.json; rm -f yarn-lock.json` just to be sure...

Reply:
updated to add the super nuclear option, removing lock files